tests/test_shells.py

```diff
@@ -4,9 +4,8 @@
 import numpy as np
 import pytest
 
-from cosmology import Cosmology
-
 import glass
+from glass.cosmology import Cosmology
```

Comment:
Can we move this in a `TYPE_CHECKING` block? (will need `from __future__ import annotations`)

Reply:
Good idea